src/app.js

```diff
@@ -1,58 +1,75 @@
 // Imports
-import express, { json } from "express";
+import express from "express";
 import cors from "cors";
-import bodyParser from "body-parser";
-const { urlencoded } = bodyParser;
-import { serve, setup } from 'swagger-ui-express';
-import morgan from 'morgan';
-import i18next from './middleware/i18next.js';
-import middleware from 'i18next-http-middleware';
-import swaggerOptions from '../docs/swagger.js';
-import swaggerJSDoc from 'swagger-jsdoc';
-import dotenv from 'dotenv';
+import morgan from "morgan";
+import session from "express-session";
+import passport from "passport";
+import i18next from "./middleware/i18next.js";
+import middleware from "i18next-http-middleware";
+import swaggerJSDoc from "swagger-jsdoc";
+import { serve, setup } from "swagger-ui-express";
+import dotenv from "dotenv";
+import  db from '../src/database/models/index.js'
+import swaggerOptions from "../docs/swagger.js";
+import welcomeRoute from "./routes/welcome.js";
+import authRoute from "./routes/authRoutes.js";
+// Configuration
 dotenv.config();
-// Sequelize set-up
-import Sequelize from 'sequelize';
-const sequelize = new Sequelize(process.env.DATABASE_URL);
+console.log(process.env.DATABASE_URL);
```

Comment:
I think this console was for debugging purposes, we can omit it since we don't need it